### DISCUSSION TITLE
Pytanie jaki header tag użyć

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#420] [Codding]
Jak jest ten popup to na header powinienem użyć `h1`? 
![image](https://user-images.githubusercontent.com/43492764/126869996-d2af9490-44d9-45b1-a2b7-5c51feef3cd2.png)

Nie wiem też, czy dobrze robię z `font-size`, powinienem tak tutaj sobie zmieniać ten `font-size` dla komponentu `Header`. Czy to jakoś globalnie powinno się dziać?
![image](https://user-images.githubusercontent.com/43492764/126870028-4995ad0b-be7d-49ea-a2ee-56434f1ca089.png)

Bo możnaby chyba zmieniać `font-size` dla całego `body` na podstawie `@media (max-width)` i wtedy w headerach nic nie zmieniać by trzeba było. Wszystkie headery wtedy byłyby spójne chyba

**Comment by mattyl006** [comment]
h1 tylko jeden dla całej strony, h1 to chyba nazwa karty, więc w tym popupie można h2

**Comment by mattyl006** [comment]
co do fontów, to można w /utils/theme porobić zmienne do poszczególnych fontów i korzystać z tych zmiennych, tak samo jak w figmie, tylko nie wiem czy mamy ustalone stałe style dla fontów

**Reply by mattyl006** [reply]
ustawianie dla body takich rzeczy to raczej nie jest dobra praktyka, nie widziałem aby ktoś tak robił, to może mieć wpływ na to, że style będą się dłużej ładować

**Reply by itchystone** [reply]
okej 👍
